test/prob/test_fixture_cdf.hpp

```diff
@@ -526,21 +545,17 @@ class AgradCdfTestFixture : public ::testing::Test {
       Scalar3 p3_ = get_param<Scalar3>(parameters[n], 3);
       Scalar4 p4_ = get_param<Scalar4>(parameters[n], 4);
       Scalar5 p5_ = get_param<Scalar5>(parameters[n], 5);
-      vector<var> s1;
-      vector<var> s2;
-      vector<var> s3;
-      add_vars(s1, p0_, p1_, p2_, p3_, p4_, p5_);
-      add_vars(s2, p0_, p1_, p2_, p3_, p4_, p5_);
-      add_vars(s3, p0_, p1_, p2_, p3_, p4_, p5_);
+      vector<var> scalar_vars;
+      add_vars(scalar_vars, p0_, p1_, p2_, p3_, p4_, p5_);
 
-      T_return_type cdf
-          = TestClass.template cdf<Scalar0, Scalar1, Scalar2, Scalar3, Scalar4,
-                                   Scalar5>(p0_, p1_, p2_, p3_, p4_, p5_);
+      T_return_type single_cdf = pow(
```

Comment:
You'll notice a pow here. In the old version we were comparing gradients of something like `grad(x)` with the gradients of something like `grad(x^r)`. To do this there was extra compare logic in `test_multiple_gradient_values`.

As I recall this was confusing for higher order things, so I just added a pow here so we're comparing `grad(x^r)` directly against `grad(x^r)` computed another way and there's no confusion.